### DISCUSSION TITLE
Suggestion - alt text translation

### DISCUSSION THREAD

**Discussion by staringatclouds** [mastodon/mastodon#19697] [General]
Pretty much as the title says

Mastodon toots come in many languages and there is an excellent translation tool built in for them

Could this be extended in some future version to include the alt text on images so visually impaired people can read them regardless of the language used ?

**Comment by garrett** [comment]
Several universities in Europe collaborated with Mozilla with a grant from the EU to develop an on-device translation library in WASM.

It's used in an official Mozilla translation extension (and there's a fork too).

It could probably be made to work for Mastodon as well.

https://browser.mt/

https://github.com/mozilla/firefox-translations

https://github.com/jelmervdl/translatelocally-web-ext
